packages/now-cli/src/commands/domains/verify.ts

```diff
@@ -18,12 +18,16 @@ export default async function verify(
     return 1;
   }
 
-  output.error(
+  const error = new Error(
```

Comment:
You can use `NowBuildError` if you dont want to create a custom error with `action` and `link` properties